src/app/category-chart/services/data.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from "@angular/core";
+import { Observable } from "rxjs/Observable";
+
+@Injectable()
+export class DataService {
```

Comment:
Is this degree of complexity worthwhile for a simple sample? Are we trying to leverage a minimal level of best practices or something? The sample is probably easier to follow if it is simply local to the component. @kdinev  will probably have to talk about what their goals have been for the samples. Whether we are optimizing for a minimal level of best practices, or whether we are optimizing for, maybe, being able to view the whole sample in stackblitz without having to navigate between 4 or 5 different files.